core/envelope-sim/src/main/java/fr/sncf/osrd/envelope_sim/PhysicsRollingStock.java

```diff
@@ -53,9 +53,6 @@ static double getMaxEffort(double speed, TractiveEffortPoint[] tractiveEffortCur
     /** The maximum constant deceleration, in m/s^2 */
     double getDeceleration();
 
-    /** The maximum braking force which can be applied at a given speed, in newtons */
-    double getMaxBrakingForce(double speed);
```

Comment:
@woshilapin wrote:
> The PR seems good. However, I'm missing some understanding about why it's fine to remove some (possibly not really well-supported) functionality: why removing `MAX`? It'd be nice to at least explain in the PR description why it was there? Why we decided to remove it? Just put some tracing about this decision so "future us" understand why, at this point in time, we decided to remove `MAX`.

Added this commit to actually clean the code a bit more after your review.
Also added a little word on main comment of the PR.
🙏 Thanks !

Reply:
Thank you 😄